.circleci/config.yml

```diff
@@ -91,38 +91,41 @@ commands:
             python setup.py develop
             python -c "import nltk; nltk.download('punkt')"
 
-  installtorchgpu16:
+  installtorchgpu17:
```

Comment:
let's rip all of circleci into a new PR which we can land much faster

Reply:
Yes - see https://github.com/facebookresearch/ParlAI/pull/3521